style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #f5f5f5;
+}
+
+.container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 36px;
+  color: #fff;
+  background-color: #3c3a39;
+  height: 92px;
+}
+
+.logo {
+  color: #fff;
+  font-weight: 700;
+  font-family: 'Poppins', sans-serif;
+  height: 24px;
+  text-decoration: none;
+}
+
+.section {
+  display: flex;
+  background-color: #1c1a19;
+  background-image: url(./images/bg-img.png);
```

Comment:
[ ] kindly fix the `background-image` to match the design provided.🤓(i think the path is wrong, check the images folder it doesn't contain the big-img.png)
The Figma design: 👇
![figmadesign](https://user-images.githubusercontent.com/62156607/198582272-d7910e24-b339-4e44-a184-3fc5da40771a.PNG)

Your design : 👇
![designta3ou](https://user-images.githubusercontent.com/62156607/198582302-0503d896-2bcf-4522-b89e-2e2c5a00cff6.PNG)